java/src/jmri/jmrit/operations/trains/configurexml/TrainIconXml.java

```diff
@@ -64,6 +67,7 @@ public Element store(Object o) {
      * @param o       an Editor as an Object
      */
     @Override
+    @Deprecated
```

Comment:
@DanielBoudreau Can you include a comment like "since 4.15.7" on the @Deprecated annotations so we can tell when we've passed the point when its okay to remove?

Reply:
Will do, as far as I can tell, the deprecated code in question has never been used.

Reply:
All test pass, don't want to restart, will add "since" comment on my next commit.